packages/legend-query-builder/src/components/explorer/QueryBuilderExplorerPanel.tsx

```diff
@@ -538,9 +539,18 @@ const QueryBuilderExplorerTreeNodeContainer = observer(
                   <PURE_ClassIcon />
                 </div>
               </div>
-              <div className="tree-view__node__label query-builder-explorer-tree__node__label">
+              <div className="tree-view__node__label query-builder-explorer-tree__node__label query-builder-explorer-tree__node__label--with-action">
                 {node.label}
               </div>
+              <div className="query-builder-explorer-tree__node__actions">
+                <QueryBuilderRootClassInfoTooltip
+                  _class={guaranteeNonNullable(queryBuilderState.class)}
```

Comment:
is it safe to do a `guaranteeNonNullable` here ?

Reply:
yes it is because we won't have any explorer tree if class in undefined